splunk_add_on_ucc_framework/main.py

```diff
@@ -85,6 +85,12 @@ def main(argv: Optional[Sequence[str]] = None):
         help="Python binary name to use to install requirements",
         default="python3",
     )
+    build_parser.add_argument(
+        "--openapi",
+        type=bool,
+        help="Generate OpenAPI Description document and expose in /static/openapi.json endpoint",
+        default=False,
```

Comment:
@artemrys  ,
Let's have it enabled by default from the very beginning

Reply:
I believe you made this change in the recent commit.